app/controllers/forem/api/v1/forums_controller.rb

```diff
@@ -0,0 +1,13 @@
+require_dependency "forem/application_controller"
+
+module Forem
+  module Api
+    module V1
+      class ForumsController < ApplicationController
+        load_and_authorize_resource :class => 'Forem::Forum', :only => :show
+
+        before_action { request.format = :json }
```

Comment:
I thought that this kind of request format setting could be done at the routes level but I can't find a solution that works. I tried coming up with my own and googling for one. Nothing that I tried works.

Instead, putting `respond_to :json` at the top of the controller works, but this requires the use of the `responders` gem (which is alright imo). I think this is a better method than forcing the request format this way.